packages/flutter/lib/src/rendering/list_wheel_viewport.dart

```diff
@@ -15,8 +15,37 @@ import 'viewport_offset.dart';
 
 typedef double _ChildSizingFunction(RenderBox child);
 
+/// A delegate used by [RenderListWheelViewport] to manage its children.
+///
+/// [RenderListWheelViewport] during layout will ask the delegate to create
+/// children that are visible in the viewport and remove those that are not.
+abstract class ListWheelChildManager {
+  /// The number of children that will be provided.
+  ///
+  /// If non-null, the children will have index in the range [0, childCount - 1].
```

Comment:
Rephrase this. From this interface and its users' perspective, it's not clear what 'children' is. Explain the implications for its children (e.g. asking createChild at index `childCount` will cause an error etc).

Reply:
Can you explain why it is unclear or how can it be misunderstood? I can't rephrase to avoid ambiguity if I don't know where the ambiguity is.